Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.py

```diff
@@ -5,26 +5,59 @@
 import re
 
 
-def get_plain_text(html_regex):
+def get_plain_text(html: str, replace_line_breaks: bool, trim_result: bool):
     data = ''
-    if html_regex and html_regex.group(0):
-        data = re.sub(r'<.*?>', '', html_regex.group(0))
+    if html:
+        data = re.sub(r'<\/?br\s?\/?>', '\n', html, flags=re.I) if replace_line_breaks else html
+
+        data = re.sub(r'<.*?>', '', data)
         entities = {'quot': '"', 'amp': '&', 'apos': "'", 'lt': '<', 'gt': '>', 'nbsp': ' ',
                     'copy': '(C)', 'reg': '(R)', 'tilde': '~', 'ldquo': '"', 'rdquo': '"', 'hellip': '...'}
         for e in entities:
             data = data.replace(f'&{e};', entities[e])
+
+        if trim_result:
+            data = re.sub(r'[ \t]{2,}', ' ', data)
+            data = re.sub(r'(\s*\r?\n){3,}', '\n\n', data)
+            data = data.strip()
     return data
 
 
-def text_from_html(args):
-    html = args['html']
-    html_tag = args.get('html_tag', 'body')
+def get_body(html: str, html_tag: str, allow_fallback: bool = False):
+    if html and html_tag:
+        body = re.search(fr'<{html_tag}.*/{html_tag}>', html, re.M + re.S + re.I + re.U)
+
+        if body and body.group(0):
+            return body.group(0)
+        elif allow_fallback and html_tag.lower() == 'body':
+            return html
+    return ''
+
+
+def main():
```

Comment:
```suggestion
def main():  # pragme: no cover
```
Let's add this to exclude `main` from unit-tests coverage since it's non-relevant for unit-tests, and the coverage dropped (probably due to merging the functions) as can be seen here ("Generate coverage reports"):
https://app.circleci.com/pipelines/github/demisto/content/306249/workflows/08600a31-4e0b-416f-b799-fafe00961e7a/jobs/721876